lms/djangoapps/support/models.py

```diff
@@ -0,0 +1,9 @@
+"""
+Models used to implement support related models in such as SSO History model
+"""
+
+from simple_history import register
+from social_django.models import UserSocialAuth
+
+# Registers UserSocialAuth with simple-django-history.
```

Comment:
We can add 1-2 more lines to explain why this is done, as opposed to having history field in the model.